packages/components/src/templates/next/layouts/Collection/utils.ts

```diff
@@ -1,6 +1,11 @@
+import dayjs from "dayjs"
+import customParseFormat from "dayjs/plugin/customParseFormat"
+
 import type { AppliedFilter, Filter as FilterType } from "../../types/Filter"
 import type { CollectionCardProps } from "~/interfaces"
 
+dayjs.extend(customParseFormat)
```

Comment:
feels overkill to use dayjs just to format dates. dayjs also somewhat dangerous; any `extend` call will affect every other dayjs instance.

should we use [date-fns](https://date-fns.org/v4.1.0/docs/format) instead? might make more sense (and is lighter)

Reply:
Updated in df8e63fa23a3d0ebda68d44c5e4c7b650d3158d9.